### PR TITLE
A-RemoveCar

### PR DESCRIPTION
**Pull request by tkhahns**
Added 3 main features:
- View Help
- Remove Cars
- Create Exceptions

### CODE REVIEW

File: src/main/java/parser/Parser.java [hunk 1]
```diff
@@ -26,30 +28,48 @@ public static String getUserInput(){
...
+                return false;
+            case REMOVE_CAR_COMMAND:
+                try {
+                    String carLicensePlateNumber = CarParser.parseCarLicenseForRemoval(userInput);
+                    CarList.removeCar(carLicensePlateNumber);
+                } catch (Exception e) {
+                    System.out.println(e.getMessage());
+                }
+                return false;
+            case ADD_CUSTOMER_COMMAND:
```
Comment (AaronZZ10) [inline src/main/java/parser/Parser.java:55]: duplicate customer command, must remove one

File: src/main/java/parser/Parser.java [hunk 2]
```diff
@@ -26,30 +28,48 @@ public static String getUserInput(){
...
+                Customer customer = CustomerParser.parseIntoCustomer(userInput);
+                CustomerList.addCustomer(customer);
+                return false;
+            case ADD_CAR_COMMAND:
```
Comment (AaronZZ10) [inline src/main/java/parser/Parser.java:59]: duplicate case add car command, must remove one

File: src/main/java/parser/Parser.java [hunk 3]
```diff
@@ -26,30 +28,40 @@ public static String getUserInput(){
...
+                CustomerList.printCustomers();
+                return false;
+            case REMOVE_TRANSACTION_COMMAND:
+                TransactionList.removeTransaction(userInput);
+                return false;
+            case "exit":
+                return true;
+            default:
+                throw CliRentalException.unknownCommand();
+            }
```
Comment (rexkoh425) [inline src/main/java/parser/Parser.java]: Indentation should be 4 lesser.

**Review by rexkoh425 — APPROVED** [review]
LGTM

File: src/main/java/car/CarList.java [hunk 4]
```diff
@@ -18,9 +18,25 @@ public static void addCar(Car car) {
...
+            }
+        }
+
+        // Remove the car if it exists
+        if (carToRemove != null) {
+            carsList.remove(carToRemove);
+            System.out.println("Car with license plate " + carLicensePlateNumber + " removed from list.");
+        } else {
+            System.out.println("No car found with license plate " + carLicensePlateNumber);
+        }
```
Comment (AaronZZ10) [inline src/main/java/car/CarList.java:38]: it's better to combine line 25-38 together

**Pull request merged**